docs/src/main/asciidoc/security-customization.adoc

```diff
@@ -220,6 +220,36 @@ public class DisabledAuthController extends AuthorizationController {
 }
 ----
 
+== Adding Principal Information to Logs
+
+In some cases it may be necessary to log the current Principal and other security identity details to the centralized log management system for the audit purposes.
```

Comment:
Note that you can already log this to the access log via the %u pattern. 

This is based on the issue around Opentracing, but even then this is not correct. This needs to clean up the MDC thread local when the context dissociates from the thread, or else it will leak. It also won't be propagated by MP context propagation, so it won't really work correctly in all circumstances.

Really we should just be attaching the principal name to the span to make it easier for users and to make sure it works correct.

Reply:
@stuartwdouglas I'll drop this text and remove `Fixes` note.